sb3_contrib/common/utils.py

```diff
@@ -0,0 +1,30 @@
+import torch as th
+
+
+def quantile_huber_loss(current_quantile: th.Tensor, target_quantile: th.Tensor) -> th.Tensor:
+    """
+    The quantile-regression loss, as described in the QR-DQN and TQC papers.
+    Partially taken from https://github.com/bayesgroup/tqc_pytorch
+
+    :param current_quantile: current estimate of quantile value
+    :param target_quantile: target quantile value
+    :return: the loss
+    """
+    n_quantiles = current_quantile.shape[-1]
+    tau = (th.arange(n_quantiles, device=current_quantile.device).float() + 0.5) / n_quantiles
+
+    if current_quantile.ndim == 3:
```

Comment:
TQC has more dimensions because there are several critics?

Reply:
Yes.

Reply:
then probably add a small comment next to the `For TQC`